src/applications/disability-benefits/2346/actions/index.js

```diff
@@ -27,11 +27,8 @@ export const getVeteranInformationData = () => async dispatch => {
     };
     dispatch(fetchVeteranInformation(veteranInformation));
   } catch (error) {
-    dispatch(
-      fetchVeteranInformationFailure(
-        error,
-        Sentry.captureMessage('failed to retrieve data from the api'),
-      ),
-    );
+    // eslint-disable-next-line no-unused-expressions
+    Sentry.captureMessage('failed to retrieve data from the api');
```

Comment:
moved the sentry capture message before the dispatch since `fetchVeteranInformationFailure` should only take 1 param